recipes/slncky/run_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+set -e
+
+echo "PATH is $PATH"
+# The annotations directory needs to be uploaded by the user.
+# It is too big to be uploaded automatically.
+# test -d $PREFIX/annotations
+slncky --help > /dev/null 2>&1
```

Comment:
I pushed my changes into the slncky branch. Not sure if I need to close this pull request and open another one, or if the changes are now included with this pull request.

Reply:
Seems like they are incorporated into this pull request.